stories/Snackbar/index.js

```diff
@@ -4,7 +4,7 @@ import { storiesOf } from '@storybook/react'
 import IconClose from 'emblematic-icons/svg/ClearClose24.svg'
 
 import Snackbar from '../../src/Snackbar'
-import SnackbarWithState from '../../src/Snackbar/examples/SnackbarWithState'
+import SnackbarWithState from '../Snackbar/SnackbarWithState'
```

Comment:
`SnackbarWithState` is in the same directory, it is not necessary to `..` up a level
```suggestion
import SnackbarWithState from './SnackbarWithState'
```

Reply:
You are right, @rsmelo . Consider it done. :smile: